src/storyMap/components/StoryMapsCard.js

```diff
@@ -232,7 +232,7 @@ const StoryMapsCard = ({ title, storyMaps, showCreate = true }) => {
     () =>
       showCreate && {
         label: t('storyMap.home_create'),
-        to: 'tools/story-maps/new',
+        to: 'tools/story-maps/new?source=home_page',
```

Comment:
suggestion: use `utm_source`, which is a standard.

Reply:
@paulschreiber I tried that and plausible didn't tack it automatically, so it seems wrong to use it and then add the value to a custom prop

Reply:
See https://plausible.io/blog/utm-tracking-tags

Reply:
@paulschreiber Reading that page, and doing some research it seems that utm_source is used for external traffic to clarify from where it is comming from, so I changed it to avoid using URL parameters since our app should be able to track internally what happened, let me know what do you think